src/components/App/ExpressRecorder.tsx

```diff
@@ -756,34 +812,28 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
                             {this.translator.translate("Cancel")}
                         </button>
                     )}
-                    {showUploadUI &&
-                        !doRecording &&
-                        recordedBlobs.length > 0 &&
-                        !this.uploadedOnce && (
-                            <div className={`${styles["express-recorder__bottom"]}`}>
-                                <button
-                                    className={`xr_btn xr_btn__download ${styles["bottom__btn"]} ${styles["btn__clear"]} ${styles["btn__download"]} `}
-                                    onClick={this.saveFile}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Download a Copy")}
-                                </button>
-                                <button
-                                    className={`xr_btn xr_btn__reset ${styles["bottom__btn"]} ${styles["btn__clear"]}`}
-                                    onClick={this.recordAgain}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Record Again")}
-                                </button>
-                                <button
-                                    className={`xr_btn xr_btn-primary xr_btn__save ${styles["bottom__btn"]} ${styles["btn__save"]}`}
-                                    onClick={this.initiateUpload}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Use This")}
-                                </button>
-                            </div>
-                        )}
+                    {showController && !this.uploadedOnce ? (
```

Comment:
you may move `this.uploadedOnce` variable as well to line **700**

Reply:
can you please change this?

Reply:
done